content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -212,8 +212,22 @@ The generated Prisma Client has a reduced bundle size because the entire query e
 
 You can adjust the maximum number of database connections of each Data Proxy connection pool.
 
-To do so, in the Prisma Data Platform add the `connection_limit` parameter to the database connection string of the environment in your project.
+To do so, in the Prisma Data Platform, add the `connection_limit` parameter to the database connection string of the environment in your project.
```

Comment:
I would also delete the previous sentences and turn the previous and this one into a single one:

```suggestion
You can set the maximum number of database connections per Data Proxy instance via the `connection_limit` argument in the Data Proxy connection string.
```

Reply:
Resolved [here](https://github.com/prisma/docs/pull/4776/commits/fd38ac03dc3aabbfa643c19c4e65e98dfcdb2431)